pkg/lvm/lvm2app_test.go

```diff
@@ -365,8 +257,8 @@ func TestCreateVolumeGroupInvalidName(t *testing.T) {
 		t.Fatal("Expected no volume group in response")
 	}
 	// Perform a known good operation to ensure that the error was
-	// cleared from the handle.
-	if _, err := handle.ListPhysicalVolumes(); err != nil {
+	// cleared from the
```

Comment:
do we still need this? maybe it applied when we had a lib handle that cached errors, but now that we're using CLI...

Reply:
Good spot - we don't need this anymore.